_posts/en/newsletters/2019-01-15-newsletter.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Bitcoin Optech Newsletter #29'
+permalink: /en/newsletters/2019/01/15/
+name: 2019-01-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a security upgrade for C-Lightning,
+describes a paper and additional research into wallets that accidentally
+revealed their private keys, and lists some notable code changes in
+popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Upgrade to C-Lightning 0.6.3:** this release fixes a remote DoS
```

Comment:
Consider adding a link to the github release page: https://github.com/ElementsProject/lightning/releases and the changelog: https://github.com/ElementsProject/lightning/blob/v0.6.3/CHANGELOG.md